coverage.py

```diff
@@ -1044,19 +1044,28 @@ def work(self, identifier, license_pool=None, **calculate_work_kwargs):
         LicensePool to use is provided.) This method will not create
         new LicensePools.
 
+        If the work is newly created or an existing work is not
+        presentation-ready, a new Work will be created by calling
+        LicensePool.calculate_work(). If there is an existing
+        presentation-ready work, calculate_work() will not be called;
+        instead, the work will be slated for recalculation when its
+        metadata changes through Metadata.apply().
+
+        :param calculate_work_kwargs: Keyword arguments to pass into
+            calculate_work() if and when it is called.
+
         :return: A Work, if possible. Otherwise, a CoverageFailure explaining
         why no Work could be created.
+
         """
         work = identifier.work
-        if work:
-            # There is already a Work associated with this Identifier.
-            # Return it.
-            return identifier.work
-
-        # There is no Work associated with this Identifier. This means
-        # we need to create one. Since we can only create a Work from
-        # a LicensePool, beyond this point the CoverageProvider
-        # needs to have a Collection associated with it.
```

Comment:
This comment is redundant because you can't instantiate a CollectionCoverageProvider without providing a Collection.